src/exemples/admin/configuration.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  api: {
+    baseUrl: 'http://apiplatform:8000'
```

Comment:
I prefer a public API like [this one](http://www.markus-lanthaler.com/hydra/event-api/)